src/components/themed/ShareButtons.tsx

```diff
@@ -32,7 +32,7 @@ function ShareButton(props: { text: string; onPress: () => void; icon: string })
   const styles = getStyles(theme)
 
   return (
-    <TouchableOpacity style={styles.button} onPress={onPress}>
+    <TouchableOpacity style={styles.button} onPress={() => onPress()}>
```

Comment:
This would cause unnecessary re-renders. How about

```
const handleOnPress = useHandler(() => onPress())

return (
    <TouchableOpacity style={styles.button} onPress={handleOnPress}>	    <TouchableOpacity style={styles.button} onPress={() => onPress()}>
```